pkg/util/ec2/ec2.go

```diff
@@ -169,12 +169,21 @@ func getResponse(url string) (*http.Response, error) {
 
 // IsDefaultHostname returns whether the given hostname is a default one for EC2
 func IsDefaultHostname(hostname string) bool {
+	return isDefaultHostname(hostname, config.Datadog.GetBool("ec2_use_windows_prefix_detection"))
+}
+
+// IsDefaultHostnameForSobotka returns whether the given hostname is a default one for EC2 for Sobotka
+func IsDefaultHostnameForSobotka(hostname string) bool {
```

Comment:
let's not use Sobotka in the function name, it doesn't mean anything in the context of the open-source Agent IMO :)

suggestion:
```suggestion
func IsDefaultHostnameForIntake(hostname string) bool {
```

Reply:
Renamed.